jujugui/static/gui/src/app/utils/environment-change-set.js

```diff
@@ -638,6 +661,11 @@ YUI.add('environment-change-set', function(Y) {
               delete config[key];
             }
           });
+          // Check if we have any resources, if we do, then add them.
+          const resourceIds = ghostService.get('resourceIds');
```

Comment:
I didn't have any plans on removing this. All other commands which change records have updated the corresponding record in the model. While this is adding a new record I don't think that it is to far from the conventions we've set forward. I actually think it would have been worse to create some kind of an internal ecs database to store this as we'd then have to make sure that never got out of sync.